[135] PR and issue template (#fixes 135)

w1stler opened the pull request.

w1stler commented:
I think we can go even further - prepare docs for whole organization: https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file-for-your-organization.

OtisRed commented on ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,16 @@
+Describe your issue here
+
+#### Your environment
+* version/branch
```
Since we don't use releases this is probably not necessary. If so I would rather specify whether it's development or production

OtisRed commented on ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,16 @@
+
+#### Your environment
+* version/branch
+* which browser and its version
```
I would add "(optional)" since I think the only time it makes sense to give this information is a bug report

OtisRed commented on ISSUE_TEMPLATE.md:
```diff
@@ -0,0 +1,16 @@
+* which browser and its version
+
+#### Steps to reproduce
+Tell us how to reproduce this issue
```
Since we use "describe" in Subject of the issue I would be consistent and use "describe" instead of "tell us"

OtisRed commented on PULL_REQUEST_TEMPLATE.md:
```diff
@@ -0,0 +1,20 @@
+#### Migrations:
+N/A
+
+#### Rollback:
```
I don't know what rollback is. Maybe a more clear title here?

w1stler commented:
I've moved the content of the files from `.github` (separate repository) here and moved to `docs` folder.

Side note - not sure how to add files to `.git` folder, at least from CLI. ;-)
